### ISSUE TITLE
FFI Hacking Setting Up Fail

### ISSUE THREAD

**Issue by wtao901231** [shorebirdtech/ffi_test#10]
Fork of Dart: https://github.com/shorebirdtech/dart-sdk/tree/shorebird/dev, but it's 404 not found!

**Comment by eseidel** [comment]
Yeah, sorry, this repo is very old.  We made the dart fork private a while back.  I'm going to archive this repo since we're no longer using it.
